backend/server.py

```
@@ -1512,29 +1512,42 @@ async def lifespan(app_: "FastAPI"):
-    # deduplicates in-memory jobs, but if 2100 duplicates are persisted in
-    # MongoDB (from a previous bug), force_cleanup alone won't catch them all.
     try:
-        from packages.scheduler.cleanup import nuclear_cleanup
-        nuclear = await nuclear_cleanup(get_db())
-        if nuclear.get("deduped", 0) > 0 or nuclear.get("deleted_run_once", 0) > 0 or nuclear.get("deleted_stuck", 0) > 0:
-            log.info("Startup nuclear cleanup: run_once=%d stuck=%d deduped=%d total_remaining=%d",
-                     nuclear["deleted_run_once"], nuclear["deleted_stuck"], nuclear["deduped"], nuclear["total"])
-    except Exception as exc:
-        log.warning("Startup nuclear cleanup failed (non-fatal): %s", exc)
+        import asyncio
+        asyncio.create_task(_deferred_startup_cleanup())
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)